docs/changelog.md

```diff
@@ -11,6 +11,9 @@ permalink: /changelog/
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Gecko.kt)
 * [Configuration](https://github.com/mozilla-mobile/android-components/blob/main/.config.yml)
 
+* **support-utils**
+  * 🌟️️ **Add a `PendingUtils.defaultFlags` property to ease setting PendingIntent mutability as required for Android 31+.
+
```

Comment:
nit: Extra line.

Reply:
Thanks!
Agree the diff looks a bit interesting but the end result seems like what we'd expect.
![image](https://user-images.githubusercontent.com/11428869/152739940-444ca123-ad8f-4597-9676-fceb30ee3e55.png)

Reply:
👍🏽